sjb/config/test_cases/test_branch_origin_extended_conformance_install_system_containers.yml

```diff
@@ -109,7 +124,8 @@ extensions:
         sudo cp ci-dnsmasq.conf /etc/dnsmasq.d/ci-dnsmasq.conf
         sudo systemctl restart dnsmasq
         sudo systemctl status dnsmasq
-        OPENSHIFT_SKIP_BUILD='true' KUBECONFIG=/etc/origin/master/admin.kubeconfig TEST_ONLY='true' JUNIT_REPORT='true' make test-extended SUITE=conformance
+        # run the tests with fingers crossed
+        OPENSHIFT_SKIP_BUILD='true' KUBECONFIG=/etc/origin/master/admin.kubeconfig TEST_ONLY='true' JUNIT_REPORT='true' SUITE=conformance make test-extended TEST_EXTENDED_ARGS="-ginkgo.skip=should\sanswer\sendpoint\sand\swildcard\squeries\sfor\sthe\scluster|should\sprovide\sDNS\sfor\spods\sfor\sHostname\sand\sSubdomain\sAnnotation|should\sprovide\sDNS\sfor\sservices|should\sprovide\sDNS\sfor\sthe\scluster|should\sexpose\sthe\sprofiling\sendpoints"
```

Comment:
And if system containers don't work because  of SELinux... sounds like an issue blocking prod use :)